### ISSUE TITLE
Incompatible recipes?

### ISSUE THREAD

**Issue by carrete** [radian-software/radian#504]
I just updated my local installation of radian, and when I started emacs I got:

    Warning (straight): Packages "prescient" and "selectrum-prescient" have incompatible recipes (:repo cannot be both "raxod502/prescient.el" and "radian-software/prescient.el")
    (One of the recipes must specify a unique :local-repo) Disable showing Disable logging
    Warning (straight): Packages "selectrum-prescient" and "prescient" have incompatible recipes (:repo cannot be both "radian-software/prescient.el" and "raxod502/prescient.el")
    (One of the recipes must specify a unique :local-repo) Disable showing Disable logging
    
I did delete `*selectrum*` and `*prescient*` under straight/build and straight/repos but still same warnings. I also tried `M-x straight-pull-all` and still same warnings.

**Comment by carrete** [comment]
Oh, when I ran `M-x straight-pull-all` I got a few prompts like:

<img width="646" alt="Screen Shot 2022-05-10 at 09 20 07" src="https://user-images.githubusercontent.com/19260155/167638093-18494ce3-c0b5-4bc3-a7d7-2191de6bab09.png">

I selected `d` which appears to have done the trick

**Comment by raxod502** [comment]
See also https://github.com/melpa/melpa/pull/8022, which corrects the upstream recipes. We will need to bump the MELPA revision in the Radian lockfile once that is merged.

**Comment by carrete** [comment]
Thanks! This works with the changes merged into MELPA
